### PR TITLE
Send Data cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request cleans up misc. files under Send Data (not tied to a release).

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/platform-services/threat-intelligence-indicators.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Threat intelligence indicators can help security analysts leverage a large body
 
 ### Role capabilities
 
-To use threat intelligence indicators, you must have the correct [role capabilities](/docs/manage/users-roles/roles/role-capabilities/#threat-intel). 
+To use threat intelligence indicators, you must have the correct [role capabilities](/docs/manage/users-roles/roles/role-capabilities/#threat-intel).
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:24]:
```suggestion
To use threat intelligence indicators, you must have the correct [role capabilities](/docs/manage/users-roles/roles/role-capabilities/#threat-intel). 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Add the following capabilities:
 
 To search logs that contain correlations to threat intelligence indicators, you must first ingest the indicators. You can ingest indicators using:
 * **The Threat Intelligence tab**. See [Add indicators in the Threat Intelligence tab](#add-indicators-in-the-threat-intelligence-tab).
-* **A collector**. See [STIX/TAXII 2 Client Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source). 
+* **A collector**. See [STIX/TAXII 2 Client Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source).
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:38]:
```suggestion
* **A collector**. See [STIX/TAXII 2 Client Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source). 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 3]
```diff
@@ -54,9 +54,9 @@ To access the **Threat Intelligence** tab, go to **Manage Data > Logs > Threat I
 1. **Add Indicators**. Click to upload files that [add threat intelligence indicators](#add-indicators-in-the-threat-intelligence-tab).
 1. **Actions**. Select to perform additional actions:
     * **Edit Retention Period**. Enter the length of time in days to retain expired threat intelligence indicator files. The maximum number of days is 180.
-1. **Source Name**. The source of the threat intelligence indicator file. 
+1. **Source Name**. The source of the threat intelligence indicator file.
 1. **Storage Consumed**. The amount of storage consumed by the threat intelligence indicator file.
-1. **Indicators**. The number of threat intelligence indicators included in the file. 
+1. **Indicators**. The number of threat intelligence indicators included in the file.
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:59]:
```suggestion
1. **Indicators**. The number of threat intelligence indicators included in the file. 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 4]
```diff
@@ -54,9 +54,9 @@ To access the **Threat Intelligence** tab, go to **Manage Data > Logs > Threat I
 1. **Add Indicators**. Click to upload files that [add threat intelligence indicators](#add-indicators-in-the-threat-intelligence-tab).
 1. **Actions**. Select to perform additional actions:
     * **Edit Retention Period**. Enter the length of time in days to retain expired threat intelligence indicator files. The maximum number of days is 180.
-1. **Source Name**. The source of the threat intelligence indicator file. 
+1. **Source Name**. The source of the threat intelligence indicator file.
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:57]:
```suggestion
1. **Source Name**. The source of the threat intelligence indicator file. 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 5]
```diff
@@ -73,22 +73,22 @@ You can also add threat intelligence indicators using the API or a collector. Se
 1. In Sumo Logic, go to **Manage Data > Logs > Threat Intelligence**.
 1. Click **Add Indicators**. The dialog displays. <br/><img src={useBaseUrl('img/platform-services/threat-intelligence-add-indicators.png')} alt="Add threat intelligence indicators" style={{border: '1px solid gray'}} width="500" />
 1. Select the format of the file to be uploaded:
-    * **Normalized JSON**. A normalized JSON file. 
-    * **CSV**. A comma-separated value (CSV) file. 
-    * **STIX 2.1 JSON**. A JSON file in STIX 2.1 format. When choosing this format, you must enter the name of the source in the **Source** field provided. 
+    * **Normalized JSON**. A normalized JSON file.
+    * **CSV**. A comma-separated value (CSV) file.
+    * **STIX 2.1 JSON**. A JSON file in STIX 2.1 format. When choosing this format, you must enter the name of the source in the **Source** field provided.
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:78]:
```suggestion
    * **Normalized JSON**. A normalized JSON file. 
    * **CSV**. A comma-separated value (CSV) file. 
    * **STIX 2.1 JSON**. A JSON file in STIX 2.1 format. When choosing this format, you must enter the name of the source in the **Source** field provided. 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 6]
```diff
@@ -73,22 +73,22 @@ You can also add threat intelligence indicators using the API or a collector. Se
...
 
    See [Upload formats](#upload-formats) for the format to use in the file.
-1. Click **Upload** to upload the file. 
-1. Click **Import**. 
+1. Click **Upload** to upload the file.
+1. Click **Import**.
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:82]:
```suggestion
1. Click **Upload** to upload the file. 
1. Click **Import**. 
```

File: docs/platform-services/threat-intelligence-indicators.md [hunk 7]
```diff
@@ -206,19 +206,19 @@ Parameters:
 
 ### Threat indicators in the Cloud SIEM UI
 
-An Entity can be associated with a known indicator that has a threat type attribute, either `threatType` (in normalized JSON format and CSV format), or `indicator_types` (in STIX format as [defined by indicator_types in STIX 2.1](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_cvhfwe3t9vuo)). 
+An Entity can be associated with a known indicator that has a threat type attribute, either `threatType` (in normalized JSON format and CSV format), or `indicator_types` (in STIX format as [defined by indicator_types in STIX 2.1](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_cvhfwe3t9vuo)).
```
Comment (kimsauce) [inline docs/platform-services/threat-intelligence-indicators.md:209]:
```suggestion
An Entity can be associated with a known indicator that has a threat type attribute, either `threatType` (in normalized JSON format and CSV format), or `indicator_types` (in STIX format as [defined by indicator_types in STIX 2.1](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_cvhfwe3t9vuo)). 
```

File: docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md [hunk 8]
```diff
@@ -65,10 +65,10 @@ Define variables in the `Environment` section of the CloudFormation template.
 ```
 "Environment": {
     "Variables": {
-        “SOURCE_CATEGORY_OVERRIDE”: "<insert-value-here>"
-        “SOURCE_HOST_OVERRIDE”: "<insert-value-here>"
-        “SOURCE_NAME_OVERRIDE”: "<insert-value-here>"
-        “SOURCE_FIELDS_OVERRIDE”: "<insert-value-here>"
+        "SOURCE_CATEGORY_OVERRIDE”: "<insert-value-here>"
```
Comment (kimsauce) [inline docs/send-data/collect-from-other-data-sources/amazon-cloudwatch-logs/collect-with-cloudformation-template.md:68]:
```suggestion
        "SOURCE_CATEGORY_OVERRIDE": "<insert-value-here>"
```

File: docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md [hunk 9]
```diff
@@ -80,24 +78,24 @@ Source in a `sources` JSON array.
...
-
-  * On \*nix, to point to a folder that contains JSON files that define Sources for a Collector:
-
-    `syncSources=/path/to/sources-folder`
-
-  * On Windows (note the escaped backslashes), to point to a folder that contains JSON files that define Sources for a Collector:
-
-    `syncSources=C:\\path\\to\\sources-folder\\`
+**syncSources**. Use to define the Sources to configure upon Collector registration. The Source definitions will be continuously monitored and synchronized with the Collector's configuration. 
+* On \*nix, to point to a JSON file that defines Sources for a Collector:
```
Comment (jpipkin1) [inline docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md:90]: Is "\*nix" supposed to be "Linux"?

File: docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md [hunk 10]
```diff
@@ -80,24 +78,24 @@ Source in a `sources` JSON array.
...
+   ```
+   syncSources=/path/to/sources.json
+   ```
+* On \*nix, to point to a folder that contains JSON files that define Sources for a Collector:
```
Comment (jpipkin1) [inline docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md:94]: Again, is "\*nix" supposed to be "Linux"?

**Review by jpipkin1 — APPROVED** [review]
Looks great, @kimsauce. I have only a couple of comments.

File: docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md (hunk 9 above)
Comment (kimsauce) [reply docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md:90]: Thank you for checking! I checked our old (pre-Docusaurus) site and it was also listed there as `*nix`. Google says [it's an abbreviation for Unix-like operating systems](https://en.wikipedia.org/wiki/Unix-like#:~:text=Other%20parties%20frequently%20treat%20%22Unix,Ultrix%2C%20Xenix%2C%20and%20XNU.).

File: docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md (hunk 10 above)
Comment (kimsauce) [reply docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md:94]: https://github.com/SumoLogic/sumologic-documentation/pull/3588#discussion_r1478888165

File: docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md (hunk 9 above)
Comment (jpipkin1) [reply docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md:90]: Good to know!

**Pull request merged**